src/pages/home/report/ReportActionCompose.js

```diff
@@ -285,6 +288,8 @@ class ReportActionCompose extends React.Component {
         this.setState({selection: e.nativeEvent.selection});
         if (!this.state.value || e.nativeEvent.selection.end < 1) {
             this.resetSuggestions();
+            this.shouldBlockEmojiCalc = false;
+            this.shouldBlockMentionCalc = false;
```

Comment:
The reason I added it at first is because the variables wouldn't get set back to false if the user opened the file picker when there's nothing in the composer. Because of this the suggestion list didn't open for the first `onSelectionChange`. 

Now it'll work because of the debounce, however it's safer to keep it. Do you want me to remove it?